native_client/c_speech_features/c_speech_features.c

```diff
@@ -217,7 +217,7 @@ csf_ssc(const short* aSignal, unsigned int aSignalLen, int aSampleRate,
   }
 
   // Calculate Spectral Sub-band Centroid features
-  ssc = (csf_float*)calloc(sizeof(csf_float*), n_frames * aNFilters);
+  ssc = (csf_float*)calloc(n_frames * aNFilters, sizeof(csf_float*));
```

Comment:
Looks like there's another bug here, it should be allocating elements of size  `sizeof(csf_float)` not `sizeof(csf_float*)`.

Reply:
Good catch, I went quickly this morning and just fixed the arguments order.